.gitmodules

```diff
@@ -31,12 +25,18 @@
 [submodule "src/lib/ecl"]
 	path = src/lib/ecl
 	url = https://github.com/PX4/ecl.git
-[submodule "cmake/cmake_hexagon"]
-	path = cmake/cmake_hexagon
-	url = https://github.com/ATLFlight/cmake_hexagon
 [submodule "src/drivers/gps/devices"]
 	path = src/drivers/gps/devices
 	url = https://github.com/PX4/GpsDrivers.git
 [submodule "src/lib/micro-CDR"]
 	path = src/lib/micro-CDR
 	url = https://github.com/eProsima/micro-CDR.git
+[submodule "platforms/nuttx/src/modules/uavcan/libuavcan"]
```

Comment:
@LorenzMeier  @dagar If we aren't in agreement on merging this, then I'm going to stop because it takes a HUGE amount of time to test all the build targets and get it all working.